Use system libffi on MSVC

--- a/vl-convert-rs/Cargo.toml
+++ b/vl-convert-rs/Cargo.toml
@@ -52,3 +52,8 @@ lazy_static = { workspace = true }
 [target.'cfg(target_os = "macos")'.dependencies.libffi]
 version = "^3.2.0"
 features = ["system"]
+
+# Enable system libffi on Windows for conda-build compatibility
+[target.'cfg(target_os = "windows")'.dependencies.libffi]
+version = "^3.2.0"
+features = ["system"]
